pipeline/relation_embedder/relation_embedder/scripts/run_local.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ "$#" -ne 1 ]; then
+  echo "Usage: $0 <PIPELINE_DATE>"
+  exit 1
+fi
+
+export PIPELINE_DATE=$1
+
+PROJECT_NAME="relation_embedder"
```

Comment:
This pattern is intended for re-use, in. combination with use of `template.env` and `local.docker-compose.yml` files. The intention is to work towards running any of the lambda services using the same script / pattern from the root with one script and a docker-compose.yml for all the services.